src/Middleware/CORS/src/Infrastructure/CorsPolicy.cs

```diff
@@ -30,7 +30,10 @@ public bool AllowAnyHeader
     {
         get
         {
-            if (Headers == null || Headers.Count != 1 || Headers[0] != "*")
+            if (Headers == null || Headers.Count != 1 || Headers[0] != CorsConstants.AnyHeader)
+            {
+                return false;
+            }
             {
                 return false;
             }
```

Comment:
Remove the double `{ return false; }`.

Reply:
This was Amir's mistake
Thanks for the correction and notice

Reply:
This was Copilot's mistake
Thanks for the correction and notice